supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -696,7 +731,7 @@ _Valid values are `arm64`._
 
 [[aws-local-zone]]
 ==== image:images/yes.png[yes] AWS Local Zone (noun)
-*Description*: An _Amazon Web Services (AWS) Local Zone_ represents a location that places cloud resources closer to a metropolitan AWS Region. An AWS Local Zone location is prepended by an AWS Region code that is followed by an identifier, such as `us-west-2-lax-1a`. 
+*Description*: An _Amazon Web Services (AWS) Local Zone_ represents a location that places cloud resources closer to a metropolitan AWS Region. An AWS Local Zone location is prepended by an AWS Region code that is followed by an identifier, such as `us-west-2-lax-1a`.
```

Comment:
We now have Wavelength zones:

An _Amazon Web Services (AWS) Wavelength Zone_ embeds AWS compute and storage services within the 5G network of a communication service provider (CSP). An AWS Wavelength Zone location is prepended by an AWS Region code that is followed by a location identifier, such as `us-west-2-wl1-lax-wlz-1`.

Reply:
@dfitzmau I'm not an expert on these terms, just trying to move this PR along. Are you proposing another entry?

Reply:
I'll leave this as optional, but WZs were just recently added to our customer docs for 4.15. LZs and WZs exist is the same AWS edge zone pool.